lib/api/models/metrics.ts

```diff
@@ -0,0 +1,33 @@
+import { MetricName } from "..";
+
+export const metricBgColor = (metric: MetricName): string => {
+  return metric === "reactorHotspotTemperatureC"
+    ? "bg-sensor-1"
+    : "bg-sensor-2";
+};
+
+export const metricTextColor = (metric: MetricName): string => {
+  return metric === "reactorHotspotTemperatureC"
+    ? "text-sensor-1"
+    : "text-sensor-2";
+};
+
+export const metricVarColor = (metric: MetricName): string => {
+  return metric === "reactorHotspotTemperatureC"
+    ? "var(--color-sensor-1)"
+    : "var(--color-sensor-2)";
+};
+
+export const metricLabel = (metric: MetricName): string => {
+  return metric == "reactorHotspotTemperatureC"
+    ? "°C on Hotspot" : "°C on Reactor";
+};
+
+export const fiendlyMetricName = (metric: MetricName): string => {
+  switch (metric) {
+    case "reactorOutletTemperatureC":
+      return "Reactor Outlet Temperature";
```

Comment:
You're right. Making metrics generic would allow to make this more generic as well. Maybe keeping string names in database or i18n files would help.

I would make chart colors independent on particular metric to allow chart component pick as many colors as it needs from predefined palette.